src/components/Users/CreateUserForm.tsx

```diff
@@ -68,20 +70,51 @@ export default function CreateUserForm({ onSubmitSuccess }: Props) {
       c_password: z.string(),
       first_name: z.string().min(1, t("field_required")),
       last_name: z.string().min(1, t("field_required")),
-      email: z.string().email(t("invalid_email_address")),
+      email: z.string().email(t("invalid_email")),
       phone_number: z
         .string()
-        .regex(/^\+91[0-9]{10}$/, t("phone_number_validation")),
+        .regex(/^\+91[0-9]{10}$/, t("phone_number_must_start")),
       alt_phone_number: z
         .string()
-        .regex(/^\+91[0-9]{10}$/, t("phone_number_validation"))
+        .refine(
+          (val) => val === "" || /^\+91[0-9]{10}$/.test(val),
+          t("phone_number_must_start"),
+        )
+        .transform((val) => val || undefined)
         .optional(),
       phone_number_is_whatsapp: z.boolean().default(true),
-      date_of_birth: z.string().min(1, t("field_required")),
-      gender: z.enum(["male", "female", "other"]),
-      qualification: z.string().optional(),
-      doctor_experience_commenced_on: z.string().optional(),
-      doctor_medical_council_registration: z.string().optional(),
+      date_of_birth: z
+        .string({
+          required_error: t("field_required"),
+        })
+        .refine(
+          (dob) => {
+            const date = new Date(dob);
+            const minDate = new Date();
+            minDate.setFullYear(minDate.getFullYear() - 120);
+            return date >= minDate && date <= new Date();
```

Comment:
nitpick, just easier to read/comprehend comparisons from Math perspective: `a < x < b` kinda style.

```suggestion
            return minDate <= date && date <= new Date();
```